testament/specs.nim

```diff
@@ -236,6 +236,8 @@ proc parseSpec*(filename: string): TSpec =
           when defined(freebsd): result.err = reDisabled
         of "arm64":
           when defined(arm64): result.err = reDisabled
+        of "not defined(nimhaslibffienabled)": # a bit hacky, this could be generalized somehow
```

Comment:
wait what?!

Reply:
fixed; that was the simplest way to do enable that test conditionally; I've now changed it to avoid this hack (see commit: "avoid `not defined(nimhaslibffienabled)` hack") but previous way was simpler